polaris/ocean/tasks/cosine_bell/__init__.py

```diff
@@ -69,13 +68,13 @@ class CosineBell(Task):
         Refinement type. One of 'space', 'time' or 'both' indicating both
         space and time
 
-    icosahedral : bool
-        Whether to use icosahedral, as opposed to less regular, JIGSAW meshes
+    prefix : str
+        The prefix indicating the mesh type ('icos' or 'qu')
```

Comment:
Can we indicate what this is a prefix for? Something like both the task name and the config section name, but check me on that

Reply:
It's used a bunch of different places.  It's a prefix on the mesh name, a subdirectory within the work directory, etc.  I can indicate that.

Reply:
Should be improved now.  You can take another look if you like.